.gitignore

```diff
@@ -113,3 +113,10 @@ venv.bak/
 
 # ruff
 .ruff_cache
+
+# LSP config files
+pyrightconfig.json
```

Comment:
I would suggest adding it to your `~/.gitignore` instead. I don't like to add things to ignore that's not directly related to commitizen (yep ".vscode" is kinda not my taste either)